src/Common/src/System/IO/PathInternal.Unix.cs

```diff
@@ -16,6 +16,11 @@ internal static partial class PathInternal
 
         internal static readonly int MaxComponentLength = Interop.Sys.MaxName;
 
+        internal const string ParentDirectoryPrefix = @"../";
+
+        internal static readonly StringComparison DefaultPathComparison = StringComparison.Ordinal;
```

Comment:
> Shouldn't this be OrdinalIgnoreCase on macOS?

Yes, thanks.

> As I noted yesterday, we also already have internal support for this

I'm slightly uncomfortable with case sensitivity coming from the state of the temp directory. I need to think that over a bit.